### ISSUE TITLE
specloc documentation

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/documentation-wiki#291]
I'm tuning up the georeference service for https://github.com/ArctosDB/arctos/issues/4916, some things keep popping up. I"m not sure what to do with this so I'm dumping it here.

First the usual - we have a billion ways of avoiding “No specific locality recorded.”, a fair number of them result in crazy georeferneces, that's probably been beat to death but maybe a yearly newsletter article or something is in order.

"Do not include higher geography (continent, ocean, sea, island group, island, country, state, province, county, feature) in the Specific Locality" - there are lots of these, they cause crazy errors. Maine, Maine....

<img width="736" alt="Screen Shot 2022-10-05 at 2 06 08 PM" src="https://user-images.githubusercontent.com/5720791/194163721-cd53e012-faec-4c94-82fa-41821507bd8f.png">

...is a whole lot smaller than Maine. Same as above, I'm not sure what to do about it other than make sure people know the docs exist.

The rest of that sentence - "unless it references a place-name in another geopolitical subdivision, in which case include that subdivision in parentheses. " - is I suspect wrong and horrible advice, but I'm happy to report that doesn't seem to be a common thing in the data.

Abbreviation advice should be checked.  ``1 mi. S., 1 mi. W., Ft. Collins`` fails completely, ``1 mile South, 1 mile West of Fort Collins`` is understood. At least some abbreviations work, I'm not sure where the line is.

<img width="508" alt="Screen Shot 2022-10-05 at 3 19 07 PM" src="https://user-images.githubusercontent.com/5720791/194174398-2566c5a9-3a60-4326-b3b3-33a4a29f0bd6.png">

green is ``5.3 SW Marfa`` (basically ignored everything except placename)
red is ``5.3 miles southwest of Marfa``

highway numbers (especially abbreviated) at least occasionally seem to confuse the services

elevation in specloc (esp. abbrdt.) sometimes (usually, perhaps) leads to strange places

It would be useful to work with Nelson, or just to test our documentation examples in geolocate, and to note whatever works/doesn't.

@mkoo do ya'll have internal documentation on this?

**Comment by sharpphyl** [comment]
> "Do not include higher geography (continent, ocean, sea, island group, island, country, state, province, county, feature) in the Specific Locality" - there are lots of these, they cause crazy errors. Maine, Maine....

In adopting GADM, islands, island groups, seas, oceans, continents and most lower level administrative units (e.g. counties outside of the US and Canada) were removed from higher geography and put in Specific locality.  Does your request from last October (and in the Locality documentation) still stand?

Additionally, since we only include first level administrative units in higher geography, we have "Malaga, Malaga" because it's near the municipality of Malaga in the Province of Malaga in the Andalusia Region of Spain.  Are you only concerned for the geolocate bot and does it matter if we do the geolocation?

Just FYI, in the Handbook at https://handbook.arctosdb.org/documentation/locality.html

> Descriptive data, most importantly [Specific Locality](https://handbook.arctosdb.org/documentation/specific-locality).

The link for Specific Locality hits a deadend.

**Comment by Jegelewicz** [comment]
All of this sounds like a great project for a Geography/Locality Committee. There will always be clean-up that could be tackled systematically by a devoted group of users.

> elevation in specloc (esp. abbrdt.) sometimes (usually, perhaps) leads to strange places

Seems like a neat project. First make sure the elevation is reflected in the locality, then get it out of specloc. The trouble is those localities can be hard to find since "ft." or "m" don't necessarily indicate elevation/depth AND elevation/depth in specloc is NOT always accompanied by those terms or some version of them....

**Comment by dustymc** [comment]
>Does your request from last October (and in the Locality documentation) still stand?

Yes, the only thing that's changed is what's in geography. If you say "Maine" in both most tools (and maybe humans) will still think you might mean "Maine, Maine" rather than just "Maine." If for whatever reason you're using geography that **does not** include Maine, then it should be given in specific locality.

> "Malaga, Malaga" because it's near 

At some level this is a request to feed various things like that to geolocate and document whatever works best. I've got some ideas about what's best, some of them seem to be correct and some of them lead here (and to a bunch of 'the bot did crazy things with my crazy data!' issues....).

Lacking better ideas, I'd spell that out - 'Province of Málaga, Málaga Municipality, Malaga Street, Malaga's Diner' - who knows what a machine might do with that, but a human (even one who's not familiar with the 'howto' stickynote in your lab and years after the names have all changed) can probably follow it.

>only concerned for the geolocate bot

Not if you're 100% unquestionably accurate, precise, and unambiguous, I suppose....

If you're not always all of those things then information which can be understood by machines might be useful for various reasons. (And language that's clear to machines is, unlike much of our locality data, probably also clear to humans - good descriptions are just good!)

>Geography/Locality Committee.

Yes, I think it needs that, or a dedicated/interested person, or something of the sort.
